packages/astroport-tests/src/lib.rs

```diff
@@ -11,29 +11,29 @@ pub mod controller_helper;
 #[allow(dead_code)]
 pub mod escrow_helper;
 
-pub fn mock_app() -> TerraApp {
+pub fn mock_app() -> App {
     let mut env = mock_env();
     env.block.time = Timestamp::from_seconds(EPOCH_START);
     let api = MockApi::default();
     let bank = BankKeeper::new();
     let storage = MockStorage::new();
-    let custom = TerraMock::luna_ust_case();
+    //    let custom = TerraMock::luna_ust_case();
```

Comment:
I think you can remove it completely and the line 27

Reply:
Done